frontend/src/components/channels/ChannelDropdownMenu.tsx

```diff
@@ -76,6 +78,23 @@ export function ChannelDropdownMenu({
               </DropdownMenuItem>
             </AlertDialogTrigger>
           )}
+          {channel.localSpendableBalance > 0 && (
+            <DropdownMenuItem
+              className="flex flex-row items-center gap-2 cursor-pointer"
+              onClick={() => {
+                const amount = channel.localSpendableBalance / 1000;
+                // TODO: Fetch new onchain address
```

Comment:
Is it ok to always use hub addresses? or the user can choose between a hub one and an external one (or they can pay to an external one after, but this requires an additional transaction)

Reply:
I would suggest implementing the simplest scenario and iterating afterwards based on user feedback. 

If you want to do advanced scenarios you can always resort to use a swap provider manually for now. (e.g. to withdraw to cold storage)

Reply:
Sounds good!